### ISSUE TITLE
Recommend Auction length based on Cov Pool TVL

### ISSUE THREAD

**Issue by pdyraga** [keep-network/coverage-pools#126]
Thoroughly document auction length parameter meaning and importance. Explore scenarios, propose various auction length values based on asset pool TVL.

We have already done some work [here](https://docs.google.com/spreadsheets/d/1vsMZcvRAEX180pxnxk0iC9xaL4hEw6Lx-96UI_1ZJ3A/edit?usp=sharing) but this is not something that can be passed to Community Multisig in the current state. To consider this issue as Done, the documentation needs to be comprehensive, reviewed by Community Multisig members and adjusted based on their feedback.

**Comment by dimpar** [comment]
> For 100% case:
> 
> * 0.01 TBTC lot size becomes profitable after ~73 seconds

For 100% case:
* 0.01 TBTC lot size becomes profitable after ~18 seconds

**Comment by dimpar** [comment]
> How one-week auction length looks for the case when BTC/ETH stays the same but 1 ETH = 2500 KEEP for 50% and 100% of KEEP migrated between KEEP-only pool and coverage pool.

For 50% case:
- 0.01 TBTC lot size becomes profitable after ~12 seconds
- 0.1 TBTC lot size becomes profitable after ~2 minutes
- 0.2 TBTC lot size becomes profitable after ~4 minutes
- 0.5 TBTC lot size becomes profitable after ~10 minutes
- 1 TBTC lot size becomes profitable after ~20 minutes
- 5 TBTC lot size becomes profitable after ~97 minutes
- 10 TBTC lot size becomes profitable after ~193 minutes

For 100% case:
- 0.01 TBTC lot size becomes profitable after ~6 seconds
- 0.1 TBTC lot size becomes profitable after ~1 minutes
- 0.2 TBTC lot size becomes profitable after ~2 minutes
- 0.5 TBTC lot size becomes profitable after ~5 minutes
- 1 TBTC lot size becomes profitable after ~10 minutes
- 5 TBTC lot size becomes profitable after ~49 minutes
- 10 TBTC lot size becomes profitable after ~97 minutes

**Comment by pdyraga** [comment]
Based on this information I think a one-week auction length is reasonable for the beginning. What do you think @dimpar? If we agree here, I'll update the deployment scenario and once we confirm with the community multisig they understand the spreadsheet and how the auction length works, we can close this issue.

**Comment by dimpar** [comment]
> Based on this information I think a one-week auction length is reasonable for the beginning. What do you think @dimpar? If we agree here, I'll update the deployment scenario and once we confirm with the community multisig they understand the spreadsheet and how the auction length works, we can close this issue.

Yes, I think 1 week is a reasonable length for the cov pool auction based on the current market & KEEP-only pool stage.

**Comment by pdyraga** [comment]
@dimpar Did we confirm with community multisig they understand the parameters? Are we comfortable closing this issue?

**Comment by dimpar** [comment]
I've presented a couple of parameters including auction length and pointed to our docs where those params are described. For the rest of the params that were not covered in the presentation the upgradability decision is easier so I think it can be understood based on our docs. I've also addressed some concerns on discord and there were no more questions (at least I'm not aware of any). Anyway, I'm happy to help the community with understanding params if anyone would raise a question, but I think we can close this issue now.
